src/fidesops/service/processors/post_processor_strategy/post_processor_strategy.py

```diff
@@ -10,5 +10,7 @@ def get_strategy_name(self) -> str:
         """Returns strategy name"""
 
     @abstractmethod
-    def process(self, data: Any, identity_data: Dict[str, Any] = None) -> Any:
+    def process(
+        self, data: Any, identity_data: Dict[str, Any] = None
+    ) -> Union[List[Dict[str, Any]], Dict[str, Any]]:
```

Comment:
Being more specific with return type since we know now to expect a dict or a list.